apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -143,14 +148,19 @@ def get_query_server_config(name='beeswax', connector=None):
           LOG.info("Setting up Hive with the following node {0}".format(znode))
           if zk.exists(znode):
             hiveservers = zk.get_children(znode)
-            server_to_use = 0 # if CONF.HIVE_SPREAD.get() randint(0, len(hiveservers)-1) else 0
-            cache.set("hiveserver2", json.dumps({"host": hiveservers[server_to_use].split(";")[0].split("=")[1].split(":")[0], "port": hiveservers[server_to_use].split(";")[0].split("=")[1].split(":")[1]}))
+            server_to_use = abs(hash(str(username))) % 10 % len(hiveservers)
```

Comment:
I'm curious, why is % 10 needed?

Reply:
After speaking with @romainr, we think it would be best to drop the load balancing aspect of this pull request. If you really need the feature, do it as a separate commit and put it behind a flag like the comment in https://github.com/cloudera/hue/pull/1005/files#diff-05088e6a157c3c3047871f25b14f0ff1L146.

Reply:
I just want to take the last bit, it is not necessary